api/src/pcapi/core/offers/repository.py

```diff
@@ -691,12 +704,39 @@ def get_offer_by_id(offer_id: int) -> models.Offer:
         raise exceptions.OfferNotFound()
 
 
+def _order_stocks_by(
+    query: flask_sqlalchemy.BaseQuery, order_by: str, order_by_desc: bool
+) -> flask_sqlalchemy.BaseQuery:
+    match order_by:
```

Comment:
```suggestion
 -> flask_sqlalchemy.BaseQuery:
     column: sa_orm.Mapped[int] | sa.cast[sa.Date | sa.Time, sa_orm.Mapped[datetime.datetime | None]]
     match order_by:

```

Tu n’as plus besoin des deux type: ignore avec ceci

Reply:
sinon ok pour moi